english/beach/beach.md

```diff
@@ -0,0 +1,5 @@
+You know in your heart the beach is where you wanna be. The sun, the sand, the sea and the breeze. You close your eyes and you can just imagine the beauty. Without any hesitation you head for the door.
+
+Do you:
+[Call your friends?](https://github.com/jonotko/create-your-own-adventure/blob/jono-adventure/english/go-with-friends/friends.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.